core/src/test/java/com/linecorp/armeria/client/DefaultClientRequestContextTest.java

```diff
@@ -19,7 +19,7 @@
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.mockito.Mockito.mock;
 
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
```

Comment:
I think when migrating to junit5, we should also migrate to the convention of package-private test methods. Maybe it creates too large a diff for this PR though.

Reply:
Removed the `public` modifiers from this class.